app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/documentation_type.scala.html

```diff
@@ -27,12 +27,19 @@
 @input_text(form("documentationType.previousDocGoodsItemId" ), "declare.previousDocGoodsItemId" )
 @input_text(form("documentationType.additionalInfoCode" ), "declare.additionalInfoCode" )
 @input_text(form("documentationType.additionalInfoDescription" ), "declare.additionalInfoDescription")
-@input_text(form("documentationType.additionalDocCategoryCode" ), "declare.additionalDocCategoryCode")
-@input_text(form("documentationType.additionalDocTypeCode" ), "declare.additionalDocTypeCode" )
-@input_text(form("documentationType.additionalDocId" ), "declare.additionalDocId" )
-@input_text(form("documentationType.additionalDocLPCO" ), "declare.additionalDocLPCO" )
-@input_text(form("documentationType.additionalDocName" ), "declare.additionalDocName" )
+
```

Comment:
The name of this view shouldn't include the word `type`

The CDS PDF name for section 2 is References so this view should be called `references.scala.html`

Reply:
Agree, but this follows the existing pattern. This can be done as part of a refactor after this ticket.